connector/connect/src/main/protobuf/spark/connect/base.proto

```diff
@@ -100,18 +70,135 @@ message AnalyzePlanRequest {
   // logging purposes and will not be interpreted by the server.
   optional string client_type = 4;
 
-  // (Optional) Get the explain string of the plan.
-  Explain explain = 5;
+  repeated AnalysisTask tasks = 5;
+
+  message AnalysisTask {
+    oneof task {
```

Comment:
I think we can also put catalog methods like `listTables`/`getTable` in `AnalysisTask `

Reply:
Yes that is why I actually find model each of the method as RPC because that is more closer to the nature of RPC.